src/main/cpp/subsystems/Climber.cpp

```diff
@@ -11,10 +11,19 @@ using namespace frc3512;
 using namespace frc3512::Constants::Climber;
 using namespace frc3512::Constants::Robot;
 
+Climber::Climber() {
+    // Create new NetworkTable
```

Comment:
It's not making a new network table. It's just getting a reference to the existing one.